app/controllers/jobseekers/job_applications/build_controller.rb

```diff
@@ -4,7 +4,25 @@ class Jobseekers::JobApplications::BuildController < Jobseekers::ApplicationCont
 
   steps :personal_details, :professional_status, :personal_statement, :ask_for_support, :declarations
 
-  helper_method :back_link_path, :current_step_number, :job_application
+  helper_method :back_link_path, :job_application, :process_steps, :step_current, :steps_adjust, :steps_config
+
+  def steps_config
+    {
+      personal_details: { number: 1, title: I18n.t("jobseekers.job_applications.build.personal_details.title") },
```

Comment:
As you are in the jobseekers/job_applications/build controller, you don't need to specify those, and can just say:
```
title: t("personal_details.title")
```
etc.